### ISSUE TITLE
您好，请问模型中下面的字典是用来干嘛的，好像没有被读取，是仅仅展示模型细节吗

### ISSUE THREAD

**Issue by yangninghua** [PaddlePaddle/models#2546]
```
train_parameters = {
    "input_size": [3, 224, 224],
    "input_mean": [0.485, 0.456, 0.406],
    "input_std": [0.229, 0.224, 0.225],
    "learning_strategy": {
        "name": "piecewise_decay",
        "batch_size": 256,
        "epochs": [30, 60, 90],
        "steps": [0.1, 0.01, 0.001, 0.0001]
    }
}
```

**Comment by yangninghua** [comment]
几乎每个模型都会有，但自己又看不出哪里用了，实在打扰paddle的同学了

**Comment by wzzju** [comment]
请看这里：https://github.com/PaddlePaddle/models/blob/develop/PaddleCV/image_classification/models/mobilenet.py#L40
每个模型文件中train_parameters被赋值给模型类的params属性，然后再看这里：https://github.com/PaddlePaddle/models/blob/develop/PaddleCV/image_classification/train.py#L312
这里params属性会被用到，主要用于配置优化器。

**Comment by shippingwang** [comment]
They are redundancy codes since the previous version, should remove them and add more ```args``` to decouple. For now, this block can control the learning rate value and how many epochs it will decay when using piecewise decay strategy, besides, other values except "epochs" and "steps" are useless cause of they will be covered by ```args```. In future, we will refine the architecture code.

**Comment by yangninghua** [comment]
收到，谢谢两位
